src/test/java/com/znsio/sample/e2e/screen/android/amazon/AmazonHomeScreenAndroid.java

```diff
@@ -0,0 +1,64 @@
+package com.znsio.sample.e2e.screen.android.amazon;
+
+import com.znsio.e2e.tools.Driver;
+import com.znsio.e2e.tools.Visual;
+import com.znsio.sample.e2e.entities.SAMPLE_TEST_CONTEXT;
+import com.znsio.sample.e2e.screen.amazonsearch.AmazonHomeScreen;
+import com.znsio.sample.e2e.screen.android.calculator.CalculatorScreenAndroid;
+import io.appium.java_client.AppiumDriver;
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.touch.WaitOptions;
+import io.appium.java_client.touch.offset.PointOption;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+
+import java.time.Duration;
+
+import static com.znsio.e2e.tools.Wait.waitFor;
+
+public class AmazonHomeScreenAndroid extends AmazonHomeScreen {
+
+    private final Driver driver;
+    private final Visual visually;
+    private static final String SCREEN_NAME = CalculatorScreenAndroid.class.getSimpleName();
+    private static final Logger LOGGER = Logger.getLogger(SCREEN_NAME);
+
+    private static By searchArea = By.xpath("//android.widget.EditText");
+    private static String searchButtonXpath = "//android.view.View[@resource-id = \"nav-search-form\"]//android.widget.Button";
+
```

Comment:
xpath variables naming convention is not correct.

Reply:
corrected